app/src/main/java/pl/pola_app/ui/activity/MainActivity.java

```diff
@@ -246,7 +228,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             return;
         }
         if (requestCode == TEACH_POLA) {
```

Comment:
Co ty na to by zamienić `TEACH_POLA` na `DONATE_POLA`, lub coś  bardziej związanego z aktualnym stanem :)?